pages/linux/genfstab.md

```diff
@@ -1,7 +1,7 @@
 # genfstab
 
 > Generate output suitable for addition to an fstab file.
-> More information: <https://man.archlinux.org/man/extra/arch-install-scripts/genfstab.8>.
+> More information: <https://manned.org/extra/arch-install-scripts/genfstab>.
```

Comment:
This is an Arch Linux install script, should we moved away from archlinux.org? :thinking:

Reply:
Nice catch! The link is actually incorrect, it should have been <https://manned.org/genfstab>. But regardless, yeah — I did try to leave distro-specific pages alone, but this one escaped. I'll remove it from the PR.

That said, can we tweak the description to make it more obvious that it is Arch-specific? Here are some ideas:

> ArchLinux install script to generate output suitable for addition to an fstab file.

or perhaps

> Generate output suitable for addition to an fstab file in an ArchLinux system.

WDYT?

Reply:
I prefer:
> Arch Linux install script to generate output suitable for addition to an fstab file.
(`Arch Linux` is the official name)

The same is for the `pacstrap` page. it's part of `arch-install-scripts` package.

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/5790/commits/c4c78403adc536382658fcde58d679c7cf17d160) and [done](https://github.com/tldr-pages/tldr/pull/5795).